server.py

```diff
@@ -16,22 +18,31 @@
 
 class Handler(BaseHTTPRequestHandler):
 
-    def _get_cookies(self):
+    def _get_auth_cookies(self):
         cookies = SimpleCookie(self.headers.get('Cookie'))
+        is_exist = False
 
-        if len(cookies) > 0 and int(cookies['auth'].value) > 0:
-            return True
+        try:
+            if int(cookies['auth'].value) > 0:
+                is_exist = True
+        except KeyError:
+            print(f'There is no entity with \'auth\' key')
 
-        return False
+        return is_exist
 
     def _get_template(self, page):
-        file_loader = FileSystemLoader('templates')
+        file_loader = FileSystemLoader(sep.join((path.dirname(path.abspath(__file__)), 'templates')))
         env = Environment(loader=file_loader)
         return env.get_template(page)
 
-    def _set_cookies(self, value):
+    def _set_auth_cookies(self, value):
         cookies = SimpleCookie()
+
         cookies['auth'] = value
+
+        if value == 0:
+            cookies['auth']['expires'] = 'Thu, 01 Jan 1970 00:00:00 GMT'
```

Comment:
This is correct, but **to be more explicit** I'd go on with `_remove_auth_cookies()` rather than `_set_auth_cookies(0)`. Please let me know what you think. No need to fix that.

Reply:
fixed